### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by eliekh05** [Homebrew/discussions#3087] [Tap maintenance and Homebrew development]
how to fix the following:

and before you ask if i reinstall 

i did it

<img width="568" alt="Screen Shot 2022-03-15 at 5 16 56 PM" src="https://user-images.githubusercontent.com/67870780/158410448-29db0d2b-63da-4495-a92d-d82e56422141.png">

<img width="568" alt="Screen Shot 2022-03-15 at 5 18 16 PM" src="https://user-images.githubusercontent.com/67870780/158410723-9ecad492-e789-4348-bdfb-9b2d3fc92a91.png">

<img width="568" alt="Screen Shot 2022-03-15 at 5 19 13 PM" src="https://user-images.githubusercontent.com/67870780/158410935-70303191-6385-47a1-ad7a-8bf3f89066e7.png">

**Comment by carlocab** [comment]
Please [don't use screenshots](https://meta.stackoverflow.com/a/285557) when sharing the output of commands in your terminal. They're often difficult to read and impossible to reuse (e.g. when looking up error messages). Instead, copy the text here and wrap them in three back-ticks (```) on the line before and on the line after to format them properly.

**Comment by dbe** [comment]
Looks like gtar isn't in your PATH. 

What are the outputs of the following 3 commands:
```echo $PATH```
```which gtar```
```/usr/local/bin/gtar --version```

For me, gtar is installed in /usr/local/bin. I'd guess you've nuked your $PATH env variable and it doesn't include that folder any more.

**Comment by gromgit** [comment]
The `expected program not found in PATH or not executable` error is an upstream bug which I've just reported. I'll update the `dpkg` formula with a properly-linked patch when the bug report is accepted.

However, as the install-time caveats clearly indicate:
```
==> Caveats
This installation of dpkg is not configured to install software, so
commands such as `dpkg -i`, `dpkg --configure` will fail.
```
fixing this bug will still not enable the functionality you're trying to use.

**Reply by gromgit** [reply]
PR: https://github.com/Homebrew/homebrew-core/pull/97091

**Reply by gromgit** [reply]
PR merged. @eliekh05, try the following and let me know if you still see the same error:
```
brew update
brew upgrade dpkg
sudo dpkg -i
```

**Reply by eliekh05** [reply]
it worked thank you
